addons/website/i18n/website.pot

```diff
@@ -9203,6 +9191,15 @@ msgstr ""
 msgid "The chosen name already exists"
 msgstr ""
 
+#. module: website
+#. odoo-python
+#: code:addons/website/models/res_company.py:0
+#, python-format
+msgid ""
+"The company %(company_name)s cannot be archived because it has a linked website %(website_name)s \n"
+"Change that website's company first."
+msgstr ""
```

Comment:
When adding stuff inside `.pot`, you need to limit the diff to strictly what you added. Could you remove all other diff in this file?


If you removed something (eg you replaced a term, which should almost never happen, so in the `pot` it's actually a removal and an addition), you never remove it from the `.pot`, the next master release will re-sync everything.

It's important because if you remove something from the `.pot` because the codebase doesn't need it anymore, some DBs will still use the translation, like if they COW'd a view or something.
Probably also for custos etc

tl;dr: just add the term you added in the codebase, exactly where it should be if you export it (so like here, in alphabetical order)